ginga/rv/plugins/FBrowser.py

```diff
@@ -210,20 +214,29 @@ def makelisting(self, path):
             self.treeview.set_optimal_column_widths()
             self.logger.debug("Resized columns for {0} row(s)".format(n_rows))
 
-    def get_path_from_item(self, res_dict):
-        paths = [info.path for key, info in res_dict.items()]
-        path = paths[0]
-        return path
+    def get_paths_from_item(self, res_dict):
+        return [info.path for info in res_dict.values()]
 
     def item_dblclicked_cb(self, widget, res_dict):
-        path = self.get_path_from_item(res_dict)
-        self.open_file(path)
+        paths = self.get_paths_from_item(res_dict)
+        if len(paths) > 0:
+            self.open_file(paths[0])
```

Comment:
Hmm, maybe raise an error if `len(paths) > 0`?  There probably shouldn't be more than one if double clicking a file.

Reply:
Is it possible for someone with touchscreen _and_  keyboard to "double click" two spots on the monitor at the same time? But regardless, raising exception makes sense here.